tests/test_ufrm.py

```diff
@@ -61,6 +61,12 @@ def test_ufrm_regression(self):
             args['workspace_dir'], 'flood_risk_service_Test1.shp'),
             gdal.OF_VECTOR)
         result_layer = result_vector.GetLayer()
+
+        # Check that all four expected fields are there.
+        self.assertEqual(
+            set(('aff_bld', 'serv_bld', 'rnf_rt_idx', 'rnf_rt_m3')),
+            set(field.GetName() for field in result_layer.schema))
+
         result_feature = next(result_layer)
         result_val = result_feature.GetField('serv_bld')

```

Comment:
What about checking other field values, like `aff_bld`?

Reply:
Yep!  As noted in another discussion thread, having these per-field checks would have helped prevent the issue with the missing field values for `aff_bld`.  Patched in 91bbc64d.